content/docs/studio/user-guide/billing-and-payment.md

```diff
@@ -0,0 +1,132 @@
+
+If you add members to available seats or remove members without reducing the
+number of seats in your team, there will be no impact in the cost to you.
+
+If you increase the number of seats, your account will be charged a prorated
+amount based on the time remaining in the billing period at the time each seat
+was added. For instance, if your billing period runs from the 15th of last month
+to the 14th of this month, then, if you add 2 seats on the 10th of this month,
+you will be charged for only 5 days for the 2 new seats. In the following
+billing period, the 2 additional seats will be taken into account for the entire
+billing period.
+
+If you reduce the number of seats, you will accordingly receive an account
+credit that you can apply on your next payment.
+
+## Downgrading a plan
+
+At any point during your subscription, you can reduce the number of seats in
+your plan.
+
+To cancel the Team plan and move to the Free plan, reduce the number of seats in
+the team to 1 or 2 and click on Downgrade. When you reduce the number of seats
+in the team, a credit balance will be computed based on the number of days
+remaining in the billing period. This credit balance can be used in future
+payments.
+
+When you downgrade from the Team plan to the Free plan, you will receive a
+refund based on the number of days remaining in the billing period. In case of
+any discrepancies in the refunded amount, please
+[contact us](mailto:support@iterative.ai).
+
+To upgrade to or downgrade from an Enterprise plan, please contact us.
+
+## Billing period, auto-renewal of team plan, and grace period for non-payment
+
+The billing period for each team plan subscription is one month (30 days) long,
+and starts on the date of the subscription.
+
+All Team plan subscriptions are renewed automatically at the end of the billing
+period.
+
+Seven (7) days before the end of your billing period (month), you will receive
+an email notification indicating that the end date is approaching. You should
+maintain a valid payment method so that we can process your payment and
+auto-renew your subscription at the end of the billing period.
+
+At the end of your billing period, if we were unable to renew your payment, you
+will enter a grace period of 5 days, during which you must ensure payment.
+During the grace period, you will continue to be able to use Studio in full
+capacity. After the grace period expires, you and all your team members will
+lose access to your team's views. You will continue to have access to your
+team's Settings page. The access to the team's views will be restored once you
+clear the pending payment.
+
+Your account's primary email address will be notified after each failed payment.
+Payments may be retried up to 4 times within the next billing period. After
+this, if the payment does not succeed, you will lose access to your team's
+views. To make the payment and restore access, please contact us.
```

Comment:
They can actually restore the access from Team Settings page by clicking "Add payment method" and go through the same process as when they create a new subscription, but with some pre-populated info like a billing address and a payment method

Reply:
Changed.